### DISCUSSION TITLE
Discussion Topics for Oct 4, 2021 Engineering Sync

### DISCUSSION THREAD

**Discussion by carlosfelix2** [CDCgov/prime-reportstream#2490] [Show and tell]
Discussion Topics for Oct 4, 2021 Engineering Sync

**Comment by carlosfelix2** [comment]
1. @carlosfelix2 Quick show on how to get IntelliJ to use a new library
2. @carlosfelix2 Should we really automerge?

**Comment by whytheplatypus** [comment]
Auth / security for mobile only systems.

**Comment by whytheplatypus** [comment]
Planning to plan, should we sit down and talk through our 3 or so working tracks (FHIR, Sender Configuration, Debt, ...)? What format? Who? When?
